From 48418f0cb9e56f97a17919bc88c6bfb50bc91538 Mon Sep 17 00:00:00 2001
From: Alex Stewart <alex.stewart@ni.com>
Date: Fri, 28 Jan 2022 15:32:54 -0600
Subject: [PATCH] update-ca-certificates: use -t with mktemp

The `--tempdir` long argument is not supported by the version of busybox
`mktemp` shipped with NIRLT sumo. Use the `-t` short arg instead.

Upstream-status: Not appropriate (NILRT-specific)

Signed-off-by: Alex Stewart <alex.stewart@ni.com>
---
 sbin/update-ca-certificates | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sbin/update-ca-certificates b/sbin/update-ca-certificates
index 19f8e33..3df3430 100755
--- a/sbin/update-ca-certificates
+++ b/sbin/update-ca-certificates
@@ -115,8 +115,8 @@ trap cleanup 0
 # Helper files.  (Some of them are not simple arrays because we spawn
 # subshells later on.)
 TEMPBUNDLE="${ETCCERTSDIR}/${CERTBUNDLE}.new"
-ADDED="$(mktemp --tmpdir "ca-certificates.tmp.XXXXXX")"
-REMOVED="$(mktemp --tmpdir "ca-certificates.tmp.XXXXXX")"
+ADDED="$(mktemp -t "ca-certificates.tmp.XXXXXX")"
+REMOVED="$(mktemp -t "ca-certificates.tmp.XXXXXX")"
 
 # Adds a certificate to the list of trusted ones.  This includes a symlink
 # in /etc/ssl/certs to the certificate file and its inclusion into the
